runtime/queries/b/highlights.scm

```diff
@@ -0,0 +1,187 @@
+(_
+  lhs: (linker) @comment
+  target: (linker) @markup.strong @diff.plus
+  rhs: (linker) @comment)
+
+(linker
+  target: (link_name) @markup.strong @diff.minus) @comment
+
+((identifier) @variable
+  (#set! priority 95))
+
+((identifier) @constant
+  (#set! priority 103)
+  (#match? @constant "^[A-Z0-9_\\-]+$"))
```

Comment:
Even if we use them, these should be moved to a `#lua-match?` pattern for performance reasons.